### PR TITLE
plugin support for bitlbee, and facebook plugin

### PR DESCRIPTION
**Pull request by anderspapitto**

### CODE REVIEW

**Comment by anderspapitto** [comment]
I am already running this locally

**Comment by jagajaga** [comment]
Please rename commit's names following our guide in `CONTRIBUTING.md`.

**Comment by anderspapitto** [comment]
@jagajaga I have renamed the commits

File: pkgs/applications/networking/instant-messengers/bitlbee-facebook/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ fetchurl, fetchFromGitHub, stdenv, bitlbee, autoconf, automake, libtool, pkgconfig, glib, json_glib }:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  name = "bitlbee-facebook";
```
Comment (jagajaga) [inline pkgs/applications/networking/instant-messengers/bitlbee-facebook/default.nix]: Please add version. Where version is the date of the commit you are fetching from source. In the format `YYYY-MM-DD`.

**Comment by anderspapitto** [comment]
@jagajaga added version

**Pull request merged**